### PR TITLE
remove reference to taxonomy File

### LINKED ISSUE #672: RDF model still useful?
The RDF model for ARIA creates confusion and it is unclear if it adds value. We should explore whether to deprioritize inline references or remove it altogether.

### PR DESCRIPTION
**Pull request by jnurthen**
closes #672

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/aria/pull/1070.html" title="Last updated on Oct 21, 2019, 4:57 PM UTC (2eb58cf)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/aria/1070/e310e9f...2eb58cf.html" title="Last updated on Oct 21, 2019, 4:57 PM UTC (2eb58cf)">Diff</a>

### CODE REVIEW

**Comment by marcoscaceres** [comment]
this is insufficient to close #672, I think... the PR would need to remove all mentions of RDF throughout the spec.

**Comment by marcoscaceres** [comment]
Thanks for continuing to push forward on this @jnurthen!

File: index.html [hunk 1]
```diff
@@ -273,14 +271,12 @@ <h1>Using <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr></h
...
 		<pre class="example highlight">&lt;li role="menuitem"&gt;Open file…&lt;/li&gt;</pre>
-		<p>The roles defined in this specification include a collection of <a href="#landmark_roles">document landmarks</a> and the <a href="#role_definitions"><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> role taxonomy</a>.</p>
-		<p>The roles in this <a>taxonomy</a> and their expected behaviors are modeled using <cite><a href="https://www.w3.org/TR/2004/REC-owl-features-20040210/"><abbr title="Resource Description Framework">RDF</abbr>/<abbr title="Web Ontology Language">OWL</abbr></a></cite> [[OWL-FEATURES]]. Features of the role taxonomy provide the following information for each role:</p>
+		<p>The following information is provided for each role in the Roles Model:</p>
 		<ul>
 			<li>an informative description of the role;</li>
 			<li>hierarchical information about related roles (e.g., a <rref>directory</rref> is a type of <rref>list</rref>);</li>
```
[the remark this answers is not in the document]
Comment (jnurthen) [reply index.html]: Changed to searchbox/textbox for simplicity

File: index.html [hunk 2]
```diff
@@ -273,14 +271,12 @@ <h1>Using <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr></h
...
 			<li>context of the role (e.g., a <rref>listitem</rref> is contained inside a <rref>list</rref>);</li>
 			<li>references to related concepts in other specifications;</li>
-			<li>use of <abbr title="Web Ontology Language">OWL</abbr> to provide a type hierarchy allowing for <a>semantic</a> inheritance (similar to a <a>class</a> hierarchy); and</li>
 			<li>supported <a>states</a> and <a>properties</a> for each role (e.g., a <rref>checkbox</rref> supports being checked via <sref>aria-checked</sref>).</li>
 		</ul>
```
[the remark this answers is not in the document]
Comment (jnurthen) [reply index.html]: Logged https://github.com/w3c/aria/issues/1095

File: index.html [hunk 3]
```diff
@@ -403,92 +399,63 @@ <h1>Important Terms</h1>
...
-			<dl class="runin">
-				<dt><abbr title="Resource Description Framework">RDF</abbr> Property</dt>
-				<dd>rdfs:subClassOf</dd>
-			</dl>
-			<p>The <a>role</a> that the current subclassed role extends in the <a>taxonomy</a>. This extension causes all the properties and constraints of the superclass role to propagate to the subclass role. Other than well known stable specifications, inheritance may be restricted to items defined inside this specification, so that external items cannot be changed and affect inherited <a>classes</a>.</p>
+			<p>The <a>role</a> that the current subclassed role extends in the Roles Model. This extension causes all the properties and constraints of the superclass role to propagate to the subclass role. Other than well known stable specifications, inheritance may be restricted to items defined inside this specification, so that external items cannot be changed and affect inherited <a>classes</a>.</p>
```
[the remark this answers is not in the document]
Comment (jnurthen) [reply index.html]: Logged https://github.com/w3c/aria/issues/1096

**Comment by jnurthen** [comment]
@mcking65 please re-review

**Pull request merged**